robocopkaka/shows_app
# add user resource

#### What does this PR do?
Adds a `user` resource
#### Description of Task to be completed?
 - add model and controller for user
  - setup rspec and factory_bot
  - add specs for user resource
  - add custom response method
  - add pull request template
#### How should this be manually tested?
* Send this request:

```
curl -L -X POST 'http://localhost:3000/users' \
-H 'Content-Type: application/json' \
--data-raw '{
	"email": "kachi@gmail.com"
}'
```

Should return a response like this:

```.json
{
    "data": {
        "user": {
            "id": 1,
            "email": "kachi@gmail.com"
        }
    },
    "message": "User created successfully"
}
```
#### Any background context you want to provide?
N/A
